ports/nrf/Makefile

```diff
@@ -12,24 +12,17 @@ SD_LOWER = $(shell echo $(SD) | tr '[:upper:]' '[:lower:]')
 
 # TODO: Verify that it is a valid target.
 
-ifeq ($(SD), )
-	# If the build directory is not given, make it reflect the board name.
-	BUILD ?= build-$(BOARD)
-	include ../../py/mkenv.mk
-	include boards/$(BOARD)/mpconfigboard.mk
-	-include mpconfigport.mk
+# If the build directory is not given, make it reflect the board name
+BUILD ?= $(if $(SD),build-$(BOARD)-$(SD_LOWER),build-$(BOARD))
```

Comment:
Do we really need the softdevice here? my understanding is that its fixed per board.

Reply:
I think it's useful if you want to build with and without SD without having to remove the build directory, I was actually tempted to also add the SD version in there so we could build 2.0.1 and 5.0.0 without cleaning but I think there are some .mk file dependencies that make it hard.

Reply:
SD is upgradable, it is useful, we will role out the bootloader with S132 v6 for feather52832, to be consistent with nrf52840